xbmc/video/VideoInfoTag.h

```diff
@@ -291,6 +302,7 @@ class CVideoInfoTag : public IArchivable, public ISerializable, public ISortable
 
   int m_playCount;
   CBookmark m_resumePoint;
+  static const int VALUE_NOT_SET = -10000;
```

Comment:
If we trust our code guideline, this is fine.

https://codedocs.xyz/xbmc/xbmc/code_guidelines.html#code_guidelines_5_2